psalm.xml

```diff
@@ -7,7 +7,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
-    errorBaseline="./psalm.baseline.xml"
+    errorBaseline="psalm.baseline.xml"
```

Comment:
Note: Re-generating the baseline file also updates `psalm.xml` as seen here. I'd like to keep this change so updating the baseline file will in the future only change the baseline file.